carceral-groups-website/src/authConfig.ts

```diff
@@ -59,7 +59,10 @@ export const msalConfig = {
  * https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#openid-connect-scopes
  */
 export const loginRequest = {
-    scopes: ["User.Read"]
+    scopes: ["https://graph.microsoft.com/User.Read", 
+    "https://storage.azure.com/user_impersonation",
+    // "https://carceralwebmapstorage.blob.core.windows.net/user_impersonation",
```

Comment:
Adding this scope even with it being added to the App registration at [Carceral Webmap](https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/~/Overview/appId/25010531-d3d4-42ac-b87e-ee58843cdd08/isMSAApp~/false) causes the scope to be invalid. Any thoughts on how to approach this?

Reply:
It looks good from what I can tell off the top of my head. I did pull the code and give it a try and although I didn't get the same error, I did still get a login failure. I'll take another fresh look tomorrow morning.

![image](https://github.com/DrewAGamboa/Carceral-Groups-Website/assets/23465532/f7e3ab5c-a675-46ba-a893-f3d8636b33d8)

Reply:
Thanks. I updated the PR description a bit with references on how I was calling MS Graph with the user's auth token. Maybe we need to do a similar approach instead which puts the access token directly in the Authorization header?